frontend/src/App.js

```diff
@@ -1,42 +1,30 @@
 import React, { Component } from 'react';
 import logo from './logo.png';
```

Comment:
Here you have *eslint* and *ts* errors:
![image](https://user-images.githubusercontent.com/23021242/51537982-0dbf3e00-1e1e-11e9-8c61-f41dfb851685.png)